app/controllers/admin/changeTrackVolunteer.py

```diff
@@ -0,0 +1,50 @@
+from flask import request, render_template, redirect, url_for, request, flash
+from app.controllers.admin import admin_bp
+from app.models.event import Event
+from app.models.user import User
+from app.models.eventParticipant import EventParticipant
+from app.logic.searchVolunteers import searchVolunteers
+from app.logic.updateTrackVolunteers import updateTrackVolunteers, addVolunteerToEvent
+from app.models.user import User
+from peewee import *
+
+@admin_bp.route('/searchVolunteers/<query>', methods = ['GET'])
+def searchTrackVolunteers(query):
+    '''Accepts user input and queries the database returning results that matches user search'''
+    query = query.strip()
+    search = query.upper()
+    splitSearch = search.split()
+    try:
+        return searchVolunteers(query)
+
+    except Exception as e:
+        return e, 500

```

Comment:
If you're just returning any exception that is thrown there's no reason for a try/catch